_includes/docs/0.4.x/gettingstarted.html

```diff
@@ -59,7 +59,7 @@
   <p>
     Another way to use the MEAN.JS boilerplate is to download a zip copy from the master branch on github. You can also do this using wget command:
```

Comment:
@lirantal Not trying to be nit picky :) but this sentence still says "master branch".